### PR TITLE
Fix table design on safari and mobiles

### PR DESCRIPTION
**Pull request by ia3andy**
This should be synchronized with a website PR https://github.com/quarkusio/quarkusio.github.io/pull/408/files

### CODE REVIEW

File: docs/src/main/asciidoc/javascript/config.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ if(tables){
         if (table.classList.contains('searchable')) { // activate search engine only when needed
```
Comment (gsmet) [inline docs/src/main/asciidoc/javascript/config.js]: Aren't `javascript/config.js` and `stylesheet/config.js` in this repo too?

BTW, I'm strongly against reformatting all the file. Or do it in a separate commit then add a commit with your changes.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/javascript/config.js (hunk 1 above)
Comment (FroMage) [reply docs/src/main/asciidoc/javascript/config.js]: I don't see any reformat changes. It's in the other PR, no?

**Review by FroMage — APPROVED** [review]
LGTM

File: docs/src/main/asciidoc/javascript/config.js (hunk 1 above)
Comment (gsmet) [reply docs/src/main/asciidoc/javascript/config.js]: It's in the other PR but shouldn't be. The master config.js is in this repo.

Comment (FroMage) [reply docs/src/main/asciidoc/javascript/config.js]: Well the other PR contains the copy of docs from this repo, like for all guides.

Comment (ia3andy) [reply docs/src/main/asciidoc/javascript/config.js]: Should I change something here?

**Review by gsmet — APPROVED** [review]
